### ISSUE TITLE
Flat palate

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11246]
**Preferred term label:**
Flat palate

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://uts.nlm.nih.gov/uts/umls/concept/C0431557

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://www.ncbi.nlm.nih.gov/books/NBK616087/

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001411
- label: Flat palate
- parents: [Abnormal palate morphology (HP:0000174)](https://hpo.jax.org/app/browse/term/HP:0000174)
- pmid: PMID:18402092
- def: Reduced height of the palate. A flat palate is present with a palatal index angle below 30 degrees, a normal palate with an angle between 30 and 45 degrees, and a steep palate with an angle above 45 degrees.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11246
- synonyms:
